src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTag.java

```diff
@@ -2,14 +2,14 @@
 WriteTag
 tag = @author
 tagFormat = \\S
-tagSeverity = (default)info
+tagSeverity = error
```

Comment:
Why did this change?

Reply:
Because this input started to work. And this update to make match what is in input

Reply:
this check is most bad from design perspective (design of Check) I always struggle to understand how it should work. One day we will deprecate it, but for now we support it as is, some people use it as it has feature that non of others our Check can do, I do not remember which feature.